linux.md

```diff
@@ -373,7 +373,7 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     cd ~
     mkdir -p projects
     cd projects
-    pnpm create expo@latest expo-test --template blank
+    pnpm create expo-app@latest --template blank-typescript expo-test
     cd expo-test
     echo 'node-linker=hoisted' > ./.npmrc
```

Comment:
I think we can remove this step, because the `create-expo-app` creates this file now when using `pnpm`
https://docs.expo.dev/more/create-expo/#pnpm

Reply:
> Didn't dey mean/say the file should be manually created

that's what I understood from that docs page too:

> Create `.npmrc` in your project's root directory and add the following configuration:

Reply:
Maybe @ProchaLu is referring to this (seemingly undocumented) behavior in `create-expo-app`:

- https://github.com/expo/expo/pull/28521

Reply:
I opened a PR to fix the Expo docs, if that is truly an undocumented behavior:

- https://github.com/expo/expo/pull/29884

Reply:
I wasn't clear in my initial [comment](https://github.com/upleveled/system-setup/pull/79#discussion_r1646518772).

Creating a new expo-app with `pnpm create expo-app` creates a `.npmrc` file. 

```bash
➜  pnpm create expo-app
.../190319afa8a-8af3                     |   +1 +
.../190319afa8a-8af3                     | Progress: resolved 1, reused 1, downloaded 0, added 1, done
✔ What is your app named? … my-app
✔ Downloaded and extracted project files.
> pnpm config --location project set node-linker hoisted
> pnpm install
 WARN  20 deprecated subdependencies found: @babel/plugin-proposal-async-generator-functions@7.20.7, @babel/plugin-proposal-class-properties@7.18.6, @babel/plugin-proposal-logical-assignment-operators@7.20.7, @babel/plugin-proposal-nullish-coalescing-operator@7.18.6, @babel/plugin-proposal-numeric-separator@7.18.6, @babel/plugin-proposal-object-rest-spread@7.20.7, @babel/plugin-proposal-optional-catch-binding@7.18.6, @babel/plugin-proposal-optional-chaining@7.21.0, abab@2.0.6, domexception@4.0.0, glob@6.0.4, glob@7.1.6, glob@7.2.3, inflight@1.0.6, osenv@0.1.5, querystring@0.2.1, rimraf@2.4.5, rimraf@2.6.3, rimraf@2.7.1, rimraf@3.0.2
Packages: +1233
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 1137, reused 1130, downloaded 0, added 1233, done
Done in 33.4s

✅ Your project is ready!

To run your project, navigate to the directory and run one of the following pnpm commands.

- cd my-app
- pnpm run android
- pnpm run ios
- pnpm run web
➜  cd my-app
```

Using cat, I can confirm that the file was created with `node-linker=hoisted`.

```bash
➜  my-app git:(main) cat .npmrc
node-linker=hoisted
```

![Screenshot 2024-06-19 at 19 51 09](https://github.com/upleveled/system-setup/assets/80746311/157aced7-3ef0-45a2-8d64-d8e2918b49bc)

Additionally, we can remove the line with `pnpm install --force` since we only added it, because we switched to `node-linker=hoisted` after the `node_modules` has been created.

> Create .npmrc in your project's root directory and add the following configuration: or are you talking about something else?

I think there is an error in the docs.

Reply:
I confirmed in my research for https://github.com/expo/expo/pull/29884 that this feature does currently exist in `create-expo-app`, nice 🎉

great simplification, thanks for the tip!

done in 91bc2345286a9384eb1df23789955b6430a00c03